deploy-agent/deployd/agent.py

```diff
@@ -550,6 +559,61 @@ def _update_deploy_alias(self, deploy_goal) -> None:
                 )
             )
 
+    def _send_deploy_info_metrics(self) -> None:
+        """
+        Emit deploy info metrics to the tsd server
+        """
+        epoch_in_seconds = int(time.time())
+        put_stmts = []
+        for env_name, deploy_status in self._envs.items():
+            report = deploy_status.report
+            build_info = deploy_status.build_info
+
+            if not report:
+                log.info(
+                    f"Skip deploy info metric for {env_name} due to missing report"
+                )
+                continue
+            if not build_info:
+                log.info(
+                    f"Skip deploy info metric for {env_name} due to missing build_info"
+                )
+                continue
+            if report.deployStage != DeployStage.SERVING_BUILD:
+                log.info(
+                    f"Skip deploy info metric for {env_name} for deploy stage {report.deployStage}"
+                )
+                continue
+            if not build_info.build_commit:
+                log.info(
+                    f"Skip deploy info metric for {env_name} due to missing commit"
+                )
+                continue
+
+            put_stmts.append(
+                f"put {DEPLOY_INFO_METRIC_NAME} "
+                f"{epoch_in_seconds} {DEPLOY_INFO_METRIC_VALUE} "
+                f"source=teletraan "
+                f"artifact={env_name} "
```

Comment:
i think it is better to use artifact = build_name. it is more meaningful and can be used to differenciate different deploy on the same host, and to idenfiy the commit sha it belongs to.

Reply:
Thanks, updated. Going to deploy to a test cluster

Reply:
thanks